doc/2/api/controllers/auth/create-api-key/index.md

```diff
@@ -0,0 +1,103 @@
+---
+code: true
+type: page
+title: createApiKey
+---
+
+# createApiKey
+
+Creates a new API key.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/api-keys/_create[?refresh=wait_for][&expiresIn=-1][&_id=null]
+Method: POST
+Body:
+```
+
+```js
+{
+  "description": "Sigfox callback authentication api key"
+}
+```
+
+### Other protocols
+
+```js
+{
+  "controller": "auth",
+  "action": "createApiKey",
+  "body": {
+    "description": "Sigfox callback authentication api key"
+  },
+
+  // optional arguments
+  "_id": "api-key-id",
+  "expiresIn": -1,
+  "refresh": "wait_for"
+}
+```
+
+---
+
+## Arguments
+  
+### Optional:
+
+- `expiresIn`: set the expiration duration (`-1` by default)
+  - if a raw number is provided (not enclosed between quotes), then the expiration delay is in milliseconds. Example: `86400000`
+  - if this value is a string, then its content is parsed by the [ms](https://www.npmjs.com/package/ms) library. Examples: `"6d"`, `"10h"`
+  - if `-1` is provided, the token will never expire
+- `refresh`: if set to `wait_for`, Kuzzle will not respond until the newly created API key is indexed
+- `_id`: set the API key unique ID to the provided value, instead of auto-generating a random ID
+
+---
+
+## Body properties
+
+- `description`: token description
+
+---
+
+## Response
+
+Returns an object containing the newly created API key:
+
+- `_id`: API key ID
+- `_source`: API key content
+  - `userId`: user kuid
+  - `hash`: fingerprint
```

Comment:
Hmm, it is a unique identifier for the key because API key were meant to be stored in the `User` object but since they are documents we didn't need this anymore.